demo/znkExercise/demo.js

```diff
@@ -121,11 +122,6 @@
             };
 
             setExercise('practice', '165');
-        })
-        .run(function ($rootScope, $translate) {
```

Comment:
Oded say it suppose to come from the translate.config.js in the index.html.
I have removed it once again and it work just fine.